schemas/put-artifact-response.yml

```diff
@@ -0,0 +1,13 @@
+$schema:            http://json-schema.org/draft-04/schema#
+title:              "Put Artifact Response"
+description: complete a blob artifact upload
+type:               object
+properties:
+  outcome:
+    type: string
+    enum:
+      - success
+      - failure
```

Comment:
There is no response from this end-point... If you really want one, return the JSON from the `artifact.json()` or whatever it is we report in `listArtifacts`

Reply:
ok, removed schema and stopped sending an output.  This will now send 204 on success